commands.go

```diff
@@ -441,13 +445,14 @@ func osucmd(args []string, s *discordgo.Session, m *discordgo.MessageCreate) {
 
 func leagueCommand(args []string, s *discordgo.Session, m *discordgo.MessageCreate, region region.Region) {
 	var playerName string
-	if len(args) < 2 && strings.ToLower(args[0]) != "help" {
+	cmd := strings.ToLower(args[0])
+	if len(args) < 2 && cmd != "help" && cmd != "verify" {
```

Comment:
this should all be way better once i convert over to slash commands

Reply:
Yeah this could have been done as to ifs checking for help and then verify then a simple else at the end saying not enough arguments as opposed to having a check at the beginning.